iterator/iterator.c

```diff
@@ -3809,6 +3810,9 @@ processFinished(struct module_qstate* qstate, struct iter_qstate* iq,
 	/* make sure QR flag is on */
 	iq->response->rep->flags |= BIT_QR;
 
+	/* explicitly set the EDE string size to 0 */
+	iq->response->rep->reason_bogus_str_size = 0;
```

Comment:
It is more consistent to also update the str to NULL.
```suggestion
	iq->response->rep->reason_bogus_str = NULL;
	iq->response->rep->reason_bogus_str_size = 0;
```

Reply:
This is already resolved in #759.